src/SUMMARY.md

```diff
@@ -68,6 +68,7 @@
   - [MMM01](./MMM01.md)
   - [M161](./M161.md)
   - [HuC1](./HuC1.md)
+  - [HuC-3](./HuC3.md)
```

Comment:
Maybe we could have a similar styling as HuC 1?
```suggestion
  - [HuC3](./HuC3.md)
```

Reply:
The silkscreen on the HuC1 says “HuC1A” while the silkscreen on the HuC-3 says “HuC-3”.  The difference in punctuation is intentional.